Resources/Textures/Objects/Misc/flare.rsi/meta.json

```diff
@@ -0,0 +1,126 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CCBYNA3",
```

Comment:
I'm not sure this license exists, usually its either `CC-BY-SA-3.0`

Reply:
hm, i copied that from another sprite so that's not good